packages/features/bookings/Booker/Booker.tsx

```diff
@@ -378,12 +378,12 @@ const BookerComponent = ({
               className="md:border-subtle ml-[-1px] h-full flex-shrink px-5 py-3 md:border-l lg:w-[var(--booker-main-width)]">
               <DatePicker
                 classNames={{
-                  datePickerContainer: customClassNames?.datePickerCustomClassname?.datePickerContainer,
-                  datePickerTitle: customClassNames?.datePickerCustomClassname?.datePickerTitle,
-                  datePickerDays: customClassNames?.datePickerCustomClassname?.datePickerDays,
-                  datePickerDate: customClassNames?.datePickerCustomClassname?.datePickerDate,
-                  datePickerDatesActive: customClassNames?.datePickerCustomClassname?.datePickerDatesActive,
-                  datePickerToggle: customClassNames?.datePickerCustomClassname?.datePickerToggle,
+                  datePickerContainer: customClassNames?.datePickerCustomClassnames?.datePickerContainer,
```

Comment:
In https://github.com/calcom/cal.com/pull/14562/files#diff-6d025ff4b8fafb7784a50d92d8f6f17f7dbb351b844b349af7705f872d25ca09R23, you changed `Classnames` to `ClassNames` such that the `N` is capitalized. Should do here to be consistent, no?